### PR TITLE
Added dismissible property to showDialog action

### PR DESCRIPTION
**Pull request by mehsaandev**

### CODE REVIEW

File: modules/ensemble/lib/action/dialog_actions.dart [hunk 1]
```diff
@@ -18,11 +18,13 @@ class ShowDialogAction extends EnsembleAction {
   ShowDialogAction({
     super.initiator,
     required this.body,
+    this.dismissible = true,
```
Comment (kmahmood74) [inline modules/ensemble/lib/action/dialog_actions.dart:21]: don't hardcode the default here, you already have a default when reading the payload. Bad design to hardcode in multiple places

**Review by kmahmood74 — APPROVED** [review]
approved with a minor change

**Comment by kmahmood74** [comment]
remember to - 

1. update schema
2. add this as an example to kitchen sink's existing dialog example
3. update documentation

File: modules/ensemble/lib/action/dialog_actions.dart (hunk 1 above)
Comment (mehsaandev) [reply modules/ensemble/lib/action/dialog_actions.dart]: removed hard coded default value

**Pull request merged**